src/Components/Admin/Rooms/roomcard.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import defaultImage from "./defaultImage.svg";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles(() => ({
+    card: {
+        fontSize: 16,
+    },
+    cardImage: {
+        width: "90%",
+        maxWidth: 400,
+        marginTop: 16,
```

Comment:
I am no front-end developer so would like some clarity on this. What's your thoughts on defining widths, margins and paddings in absolute values? Will this be a hindrance if we want to make things more responsive later on?

Reply:
My general rules are the margins, padding, and maxWidth/minWidth are fine to hardcode. Widths are not great to hardcode, but in this case it makes sense since it ensures the image takes up 90% of the column (100% looks bad) and should be fine with responsiveness later on.